packages/legend-application/style/components/shared/_packageable-element-format-option.scss

```diff
@@ -41,6 +41,21 @@
   }
 }
 
+.packageable-element-format-option-label-type {
+  background-color: var(--color-dark-grey-300);
+  margin: -1rem 0.8rem -1rem -1.2rem;
+  height: 3.2rem;
+  padding: 0 0 0 0.5rem;
+
+  &__system {
```

Comment:
Also, we follow [BEM](http://getbem.com/introduction/) convention, so here, I believe we should do `--system`, `--generated`

Reply:
! makes sense, switching to BEM